ui/raidboss/data/04-sb/raid/o11s.js

```diff
@@ -460,12 +461,15 @@
     {
       'locale': 'cn',
       'replaceSync': {
+        'Augmented Rocket Punch': '强化型火箭飞拳',
         'Engage!': '战斗开始！',
         'Level Checker': '等级检测仪',
         'Omega': '欧米茄',
-        'rocket punch': '火箭飞拳',
+        'Rocket Punch': '火箭飞拳',
```

Comment:
Does this need a ^ and $ on it, since it will conflict with Augmented Rocket Punch?

Reply:
in this case it would work without a problem (alphabetical order, so Augmented Rocket Punch would always comes first).